Tangem/Modules/Send/Amount/Interactor/SendAmountInteractor.swift

```diff
@@ -57,6 +57,8 @@ class CommonSendAmountInteractor {
         self.amountModifier = amountModifier
         self.type = type
 
+        _cachedAmount.send(input.amount)
```

Comment:
тут сомнительно, но проблема в том что если не заполнить кеш - подписка в 66 строке приведет к вызову `output?.amountDidChanged(amount: amount)` с нулевым амантом и инпут снаружи сбросится в ноль.

Reply:
Блин, походу возможно в Sell оторвало =/
давай тогда через `init` сделаем. А инициализацию с `nil` вообще уберем (

```suggestion
        _cachedAmount = .init(input.amount)
```

Reply:
поправил на init